basiclti/tsugi-util/src/java/org/tsugi/pox/IMSPOXRequest.java

```diff
@@ -173,7 +176,18 @@ public IMSPOXRequest(String bodyString)
 	public void loadFromRequest(HttpServletRequest request) 
 	{
 		String contentType = request.getContentType();
-		if ( ! "application/xml".equals(contentType) ) {
+		String baseContentType;
+
+		try {
+			MimeType mimeType = new MimeType(contentType);
+			baseContentType = mimeType.getBaseType();
+		} catch (MimeTypeParseException e){
+			errorMessage = "Unable to parse mime type";
+			log.info("{}\n{}", errorMessage, contentType);
+			return;
+		}
+
+		if ( ! "application/xml".equals(baseContentType) ) {
```

Comment:
I prefer using constants here instead of "application/xml", but looks good to me.